frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/vms/UnitVmModel.java

```diff
@@ -4065,7 +4065,8 @@ protected void updateCpuPinningPolicy() {
 
         boolean isDedicatedCpusSupported = false;
         if(getSelectedCluster() != null && getSelectedCluster().getCompatibilityVersion() != null) {
-            isDedicatedCpusSupported = Version.v4_7.lessOrEquals(getSelectedCluster().getCompatibilityVersion());
+            isDedicatedCpusSupported = AsyncDataProvider.getInstance().isDedicatedPolicySupportedByVersion(
+                    getSelectedCluster().getCompatibilityVersion());
```

Comment:
we should rather use `getCompatibilityVersion()` here - @liranr23  do you prefer to do it separately or as part of this PR?

Reply:
Within this one.